src/components/content/Notifications.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import styled from 'styled-components'
+
+interface User {
+  id: string
+  username: string
+}
+
+interface NotificationProp {
+  author: User
+  body: string
+  timestamp: string
+  readed: boolean
+}
+
+export default function Notifications(entries: NotificationProp[]) {
```

Comment:
Hi Kate, thank you for your contribution! To make this file a proper react function component, the argument must be an object with entries as a key, so we need `Notifications({entries} : NotificationsProps)` here, and the interface should be adapted, too.

Could you please fix this? The rest looks fine.

Reply:
fixed